Data/Enums/FamilyMember.cs

```diff
@@ -0,0 +1,14 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Data.Enums
+{
+    public enum FamilyMember
```

Comment:
Please ensure all enums have the suffice "Enum"  eg...............FamilyMemberEnum